roles/kubernetes/node/defaults/main.yml

```diff
@@ -35,12 +35,29 @@ kubelet_disable_shared_pid: true
 ### fail with swap on (default true)
 kubelet_fail_swap_on: true
 
-# Reserve this space for kube resources
-kube_memory_reserved: 256M
-kube_cpu_reserved: 100m
-# Reservation for master hosts
-kube_master_memory_reserved: 512M
+# kubelet Reservation for node hosts
+kube_node_memory_reserved: 256Mi
+kube_node_cpu_reserved: 100m
+# kubelet Reservation for master hosts
+kube_master_memory_reserved: 512Mi
 kube_master_cpu_reserved: 200m
+# kubelet Reserved Cgroups
+kube_reserved_cgroups: /kubereserved
```

Comment:
You need to reconfigure docker daemon and kubelet to run in this cgroup for this kubelet argument to have any effect.

Reply:
All relevant services need to be run in that cgroup or else that is just an empty group.